freediscovery/server/resources.py

```diff
@@ -896,3 +898,29 @@ def post(self, **args):
             res = sorted(res, key=lambda row: row['score'], reverse=True)
 
         return {'data': res}
+
+
+# ============================================================================ #
+#                            Custom Stop Words
+# ============================================================================ #
+
+
+class CustomStopWordsApi(Resource):
+    @doc(description="Perform a mechanism for adding / managing custom stop words")
+    @use_args({"name" : wfields.Str(required=True),
+               "stop_words" : wfields.List(wfields.Str(), required=True)})
+    @marshal_with(CustomStopWordsSchema())
+    def post(self, **args):
+        name = args['name']
+        stop_words = args['stop_words']
+        model = _StopWordsWrapper(cache_dir=self._cache_dir)
+        model.save(name = name, stop_words = stop_words)
+        return {'name': name}
+
+
+class CustomStopWordsLoadApi(Resource):
+    @doc(description="Loading a custom stop words")
+    @marshal_with(CustomStopWordsLoadSchema())
+    def get(self, **args):
+        name = args['name']
```

Comment:
This could just be,
```py
def get(self, name):
```
there is no other parameters, and the `name` parameter is parsed from `'/stop-words/<name>`